commit bf3605187530d82ece4f22bb1f2ac53d231f41d4
Author: Tim Harvey <tharvey@gateworks.com>
Date:   Tue Mar 30 08:19:43 2021 -0700

    dt-bindings: arm: imx: add imx8mm gw7901 support
    
    The Gateworks GW7901 is an ARM based single board computer (SBC)
    featuring:
     - i.MX8M Mini SoC
     - LPDDR4 DRAM
     - eMMC FLASH
     - SPI FRAM
     - Gateworks System Controller (GSC)
     - Atmel ATECC Crypto Authentication
     - USB 2.0
     - Microchip GbE Switch
     - Multiple multi-protocol RS232/RS485/RS422 Serial ports
     - onboard 802.11ac WiFi / BT
     - microSD socket
     - miniPCIe socket with PCIe, USB 2.0 and dual SIM sockets
     - Wide range DC power input
     - 802.3at PoE
    
    Acked-by: Rob Herring <robh@kernel.org>
    Signed-off-by: Tim Harvey <tharvey@gateworks.com>
    Reviewed-by: Fabio Estevam <festevam@gmail.com>
    Signed-off-by: Shawn Guo <shawnguo@kernel.org>

diff --git a/Documentation/devicetree/bindings/arm/fsl.yaml b/Documentation/devicetree/bindings/arm/fsl.yaml
index e3c50f231d71..eacbc1f8d466 100644
--- a/Documentation/devicetree/bindings/arm/fsl.yaml
+++ b/Documentation/devicetree/bindings/arm/fsl.yaml
@@ -685,6 +685,7 @@ properties:
               - gw,imx8mm-gw71xx-0x       # i.MX8MM Gateworks Development Kit
               - gw,imx8mm-gw72xx-0x       # i.MX8MM Gateworks Development Kit
               - gw,imx8mm-gw73xx-0x       # i.MX8MM Gateworks Development Kit
+              - gw,imx8mm-gw7901          # i.MX8MM Gateworks Board
               - kontron,imx8mm-n801x-som  # i.MX8MM Kontron SL (N801X) SOM
               - variscite,var-som-mx8mm   # i.MX8MM Variscite VAR-SOM-MX8MM module
           - const: fsl,imx8mm